plugins/auth-backend-module-bitbucket-server-provider/api-report.md

```diff
@@ -0,0 +1,45 @@
+## API Report File for "@backstage/plugin-auth-backend-module-bitbucket-server-provider"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { BackendFeature } from '@backstage/backend-plugin-api';
+import { OAuthAuthenticator } from '@backstage/plugin-auth-node';
+import { PassportOAuthAuthenticatorHelper } from '@backstage/plugin-auth-node';
+import { PassportProfile } from '@backstage/plugin-auth-node';
+import { SignInResolverFactory } from '@backstage/plugin-auth-node';
+
+// @public (undocumented)
+const authModuleBitbucketServerProvider: BackendFeature;
+export default authModuleBitbucketServerProvider;
+
+// @public (undocumented)
+export const bitbucketServerAuthenticator: OAuthAuthenticator<
+  {
+    helper: PassportOAuthAuthenticatorHelper;
+    host: string;
+  },
+  PassportProfile
+>;
+
+// @public (undocumented)
+export type BitbucketServerOAuthResult = {
+  fullProfile: PassportProfile;
+  params: {
+    scope: string;
+    access_token?: string;
+    token_type?: string;
+    expires_in?: number;
+  };
+  accessToken: string;
+  refreshToken?: string;
+};
+
+// @public
+export namespace bitbucketServerSignInResolvers {
+  const emailMatchingUserEntityProfileEmail: SignInResolverFactory<
+    BitbucketServerOAuthResult,
```

Comment:
This doesn't match the authenticator itself, which ends up using `OAuthAuthenticatorResult<PassportProfile>`. The `BitbucketServerOAuthResult` type also contains the same information as the new `OAuthAuthenticatorResult` so I think we should be able to get rid of it.

Reply:
Fixed 🙏🏻